client/landing/jetpack-cloud/sections/backups/main.jsx

```diff
@@ -183,7 +181,7 @@ class BackupsPage extends Component {
 					siteSlug={ siteSlug }
 				/>
 
-				<div>{ isLoadingBackups && 'Loading backups...' }</div>
+				<div>{ isLoadingBackups && translate( 'Loading backups…' ) }</div>
```

Comment:
This message was temporal. We could remove it and substitute it with a better design while we are waiting as Calypso does in other sections.